test/integration/test_pulsar_embedded_metadata.py

```diff
@@ -16,6 +16,10 @@ class EmbeddedMetadataPulsarIntegrationInstance(integration_util.IntegrationInst
     @classmethod
     def handle_galaxy_config_kwds(cls, config):
         config["job_config_file"] = EMBEDDED_PULSAR_JOB_CONFIG_FILE
+        config['object_store_store_by'] = 'uuid'
+        # metadata_strategy is being set to directory in embedded_pulsar_metadata_job_conf.yml,
+        # since extended_metadata doesn't work yet on pulsar
+        config['metadata_strategy'] = 'extended'
```

Comment:
What are you testing here? Shouldn't this fail because pulsar does not work yet?

Reply:
I tried to explain that in the comment above, this tests that the `metadata_strategy` can be overriden in the destination.

Reply:
Yeah, but once this is fixed the test will not test this anymore, correct? Or in other words, once pulsar is capable of extended-metadata, we do not test the overwriting anymore. Should we worry about this?

Reply:
I don't think we should at this point. I can move this to a separate test if you like.